docs/guides/prisma.md

```diff
@@ -0,0 +1,360 @@
+# Adding Prisma with Platformatic
+
+[Prisma](https://www.prisma.io/) is an open-source ORM for Node.js and TypeScript. It is used as an alternative to writing SQL, or using another database access tool such as SQL query builders (like [knex.js](https://knexjs.org/)) or ORMs (like [TypeORM](https://typeorm.io/) and [Sequelize](https://sequelize.org/)). Prisma currently supports PostgreSQL, MySQL, SQL Server, SQLite, MongoDB, and CockroachDB.
+
+Prisma can be used with JavaScript or TypeScript, and provides a level to type-safety that goes beyond the guarantees made by other ORMs in the TypeScript ecosystem. You can find an in-depth comparison of Prisma against other ORMs [here](https://www.prisma.io/docs/concepts/more/comparisons).
+
+If you want to get a quick overview of how Prisma works, you can follow the [Quickstart](https://www.prisma.io/docs/getting-started/quickstart) or read the [Introduction](https://www.prisma.io/docs/understand-prisma/introduction) in the Prisma documentation. 
+
+
+## How Prisma can improve your workflow with Platformatic DB:
+
+While Platformatic speeds up development of your REST and GraphQL APIs, Prisma can complement the workflow in several ways:
+
+1. Provides an intuitive data modeling language
+1. Provides auto-generated and customizable SQL migrations
+1. Provides type-safety and auto-completion for your database queries
+
+
+You can learn more about why Prisma and Platformatic are a great match [this article](https://dev.to/prisma/why-prisma-and-platformatic-are-a-great-match-2dkl).
+
+## Prerequisites
+
+To follow along with this guide, you will need to have the following:
+- [Node.js](https://nodejs.org/) >= v16.17.0 or >= v18.8.0
+- [npm](https://docs.npmjs.com/cli/) v7 or later
+- A code editor, for example [Visual Studio Code](https://code.visualstudio.com/)
+- A Platformatic DB project
+
+## Setup Prisma
+
+Install the [Prisma CLI](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-cli) and the [`db-diff`](https://github.com/ruheni/db-diff) development dependencies in your project:
+
+```bash
+npm install --save-dev prisma @ruheni/db-diff
+```
+
+Next, initialize Prisma in your project
+
+```bash
+npx prisma init
+```
+
+This command does the following:
+
+- Creates a new directory called `prisma` which contains a file called `schema.prisma`. This file defines your database connection and the Prisma Client generator.
+- Creates a `.env` file at the root of your project if it doesn't exist. This defines your environment variables (used for your database connection).
+
+You can specify your preferred database provider using the `--datasource-provider` flag, followed by the name of the provider: 
+
+```bash
+npx prisma init --datasource-provider mysql # or sqlite, sqlserver, cockroachdb
+```
```

Comment:
As the default for `prisma init` is `postgres`, I would suggest using this command above instead.